bootstrap/eksctl/README.md

```diff
@@ -163,6 +167,35 @@ kubectl wait --for condition=established --timeout=300s crd/providerconfigs.kube
 kubectl apply -f crossplane/kubernetes-provider-config.yaml
 ```
 
+### Install Crossplane Helm Provider (required for examples that deploy Helm charts)
```

Comment:
This can go under the section above with the rest of providers.

Reply:
Agree. Will move up with other provider installs.

Reply:
change made